README.md

```diff
@@ -0,0 +1,400 @@
+              "summary": "Clear",
+              "icon": "clear-night",
+              "precipIntensity": 0,
+              "precipProbability": 0,
+              "temperature": 71.13,
+              "apparentTemperature": 71.91,
+              "dewPoint": 66.23,
+              "humidity": 0.85,
+              "pressure": 1012.13,
+              "windSpeed": 2.42,
+              "windGust": 7.51,
+              "windBearing": 166,
+              "cloudCover": 0,
+              "uvIndex": 0,
+              "visibility": 9.07,
+              "ozone": 314.95
+          },
+          {...}
+      ]    
+  },
+  "daily": {
+      "summary": "Rain tomorrow through Saturday, with high temperatures falling to 72°F on Friday.",
+      "icon": "rain",
+      "data": [
+          {
+              "time": 1557032400,
+              "summary": "Partly cloudy starting in the afternoon, continuing until evening.",
+              "icon": "partly-cloudy-day",
+              "sunriseTime": 1557056731,
+              "sunsetTime": 1557105141,
+              "moonPhase": 0.03,
+              "precipIntensity": 0,
+              "precipIntensityMax": 0.0002,
+              "precipIntensityMaxTime": 1557046800,
+              "precipProbability": 0,
+              "temperatureHigh": 84.25,
+              "temperatureHighTime": 1557093600,
+              "temperatureLow": 65.1,
+              "temperatureLowTime": 1557140400,
+              "apparentTemperatureHigh": 85.89,
+              "apparentTemperatureHighTime": 1557097200,
+              "apparentTemperatureLow": 65.84,
+              "apparentTemperatureLowTime": 1557140400,
+              "dewPoint": 63.49,
+              "humidity": 0.77,
+              "pressure": 1011.49,
+              "windSpeed": 2.53,
+              "windGust": 13.3,
+              "windGustTime": 1557100800,
+              "windBearing": 132,
+              "cloudCover": 0.22,
+              "uvIndex": 7,
+              "uvIndexTime": 1557082800,
+              "visibility": 9.67,
+              "ozone": 309.52,
+              "temperatureMin": 58.2,
+              "temperatureMinTime": 1557057600,
+              "temperatureMax": 84.25,
+              "temperatureMaxTime": 1557093600,
+              "apparentTemperatureMin": 58.29,
+              "apparentTemperatureMinTime": 1557057600,
+              "apparentTemperatureMax": 85.89,
+              "apparentTemperatureMaxTime": 1557097200
+          },
+          {...},
+      ]
+   }
+}
+```
+
+In the event of a bad request, the application responds with a 401 status, and a message that the request failed to include a location and/or API key.
+
+``` HTTP
+status: 401
+body:
+
+{
+  "error": "Missing location and/or API key."
+}
+```
+
+###### Creating favorites
+
+The application supports the creating of favorite locations for authenticated users via a `POST` request to the `/api/v1/favorites` endpoint. This request must include an API key and a location in the body.
```

Comment:
Wording sounds a little funny. I think "supports the _creation_ of favorite locations" or "supports creating favorite locations" may be better.

Reply:
I opted for "the creation of" you suggested. I think it's much improved.